staging/provisioning/windows/kubeproxystart.ps1

```diff
@@ -17,10 +18,13 @@ while (!$hnsNetwork) {
     $hnsNetwork = Get-HnsNetwork | ? Name -EQ $KubeNetwork
 }
 
-# add dualstack feature gate if dualstack enabled
-$isDualStackEnabled = ("IPv6DualStack=true" | ? { $Global:ClusterConfiguration.Kubernetes.Kubelet.ConfigArgs -match $_ }) -ne $null
-if ($isDualStackEnabled) {
-    $global:KubeproxyArgList += @("--feature-gates=IPv6DualStack=true")
+# enable WinDsr if WinDsr feature gate is enabled
+if ($KubeproxyFeatureGates -contains "WinDSR=true") {
+    $global:KubeproxyArgList += @("--enable-dsr=true")
+}
+
+if ($KubeproxyFeatureGates.Count -ne 0) {
+    $global:KubeproxyArgList += @("--feature-gates='" + ($KubeproxyFeatureGates -join "','") + "'")
```

Comment:
Isn't `kubeproxyFeatureGates` already in in the format of "key=value," from `GetKubeProxyFeatureGatesWindowsArguments` (https://github.com/Azure/aks-engine/pull/4104/files#diff-dfee74d02fd350fb1ee1ffbbbc32acf4cb075c9ae42b6e31c8c7acefe7fed623R1164)

Reply:
$KubeproxyFeatureGates is a list in https://github.com/Azure/aks-engine/pull/4104/files?file-filters%5B%5D=.go&file-filters%5B%5D=.ps1#diff-bb0963ea67ff5314b6c48f54d3bb555f57341f0adc5f4d33f05e536214eda1e6R106. I followed the  same logic as `KubeletConfigArgs`